INFO: Starting training:
        Epochs:          2
        Batch size:      8
        Learning rate:   1e-05
        Training size:   3883
        Validation size: 431
        Checkpoints:     True
        Device:          cuda
        Images scaling:  0.5
        Mixed Precision: False




Epoch 1/2:  10%|████████████████▍                                                                                                                                                     | 384/3883 [00:10<01:17, 45.33img/s, loss (batch)=1.59]



INFO: Validation Hausdorff score: 1.5058773722092786



Epoch 1/2:  20%|████████████████████████████████▊                                                                                                                                     | 768/3883 [00:26<01:07, 46.48img/s, loss (batch)=1.37]


INFO: Validation Hausdorff score: 0.37936450908608654



Epoch 1/2:  30%|████████████████████████████████████████████████▉                                                                                                                    | 1152/3883 [00:41<00:58, 46.90img/s, loss (batch)=1.26]



INFO: Validation Hausdorff score: 0.1612500287802376



Epoch 1/2:  40%|█████████████████████████████████████████████████████████████████▎                                                                                                   | 1536/3883 [00:56<00:51, 45.91img/s, loss (batch)=1.18]



INFO: Validation Hausdorff score: 0.15559238503808692



Epoch 1/2:  49%|████████████████████████████████████████████████████████████████████████████████▌                                                                                    | 1896/3883 [01:13<01:17, 25.67img/s, loss (batch)=1.12]
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/queues.py", line 251, in _feed
    send_bytes(obj)
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/queues.py", line 251, in _feed
    send_bytes(obj)
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 205, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 205, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 416, in _send_bytes
    self._send(header + buf)
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 416, in _send_bytes
    self._send(header + buf)
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 373, in _send
    n = write(self._handle, buf)
  File "/home/nicolasaidoune/miniconda3/lib/python3.9/multiprocessing/connection.py", line 373, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
BrokenPipeError: [Errno 32] Broken pipe
INFO: Saved interrupt
Traceback (most recent call last):
  File "/mnt/c/Users/nicoa/OneDrive/Bureau/Projet_3A/SegmentationMain/U-Net/train.py", line 191, in <module>
    train_net(net=net,
  File "/mnt/c/Users/nicoa/OneDrive/Bureau/Projet_3A/SegmentationMain/U-Net/train.py", line 107, in train_net
    epoch_loss += loss.item()
KeyboardInterrupt